server/templates/pages/src/index.html

```diff
@@ -9,6 +9,7 @@
         <meta name="viewport" content="width=device-width">
         <meta name="referrer" content="origin">
         <meta name="robots" content="noindex,nofollow">
+        <meta name="fxa-content-server/config" content="{{ config }}" />
```

Comment:
when I run this with production config (against circle), I get ` <meta name=fxa-content-server/config content=""> `

example: build and run `CONFIG_FILES=server/config/local.json,server/config/production.json,tests/ci/config_circleci.json node_modules/.bin/grunt serverproc:dist`

output:

```
<!DOCTYPE html><!--[if lt IE 10]>      <html class="lt-ie10" dir="ltr" lang="en"> <![endif]--><!--[if gte IE 10]><!--> <html dir=ltr lang=en><!--<![endif]--> <head> <meta charset=utf-8> <meta http-equiv=X-UA-Compatible content="IE=edge"> <title>Firefox Accounts</title> <meta name=description content=""> <meta name=viewport content="width=device-width"> <meta name=referrer content=origin> <meta name=robots content=noindex,nofollow> <meta name=fxa-content-server/config content=""> <link rel=stylesheet href="http://127.0.0.1:3030/styles/9669d56c.en.css" integrity="sha512-wt5yv7xY8IyZN7K7OAhCH4MLHoSZKM8WNSyxFWySILCNL99fqWexCN8KNZbukigmh4QNxLN6CtK8j4ZpLsDqww==" crossorigin=anonymous> <script src="http://127.0.0.1:3030/scripts/3f80b962.head.js" integrity="sha512-znK0qshAgi1SasO6Q4bLnAZiEXXDCyHCKseNwYnDCgZpTi4XpqB6oxJuwwCpzRHRiZNc3ijWiR5wKPLY0GHyXg==" crossorigin=anonymous></script><!--[if lt IE 9]>
```